lib/src/customization/calendar_builders.dart

```diff
@@ -61,6 +65,10 @@ class CalendarBuilders {
   /// Mutually exclusive with `markersBuilder`.
   final SingleMarkerBuilder singleMarkerBuilder;
 
+  /// Custom builder for weekday names. They are displayed below `HeaderRow` and above calendar days.
+  /// Will overwrite `weekdayStyle` and `weekendStyle` from `DaysOfWeekStyle`.
+  final WeekdayBuilder weekdayBuilder;
```

Comment:
Lets name this `dowBuilder` to keep consistent naming.

I think it also might be beneficial to separate week days from weekends. Can you add a second builder?
Naming could be then as follows: `dowWeekdayBuilder` and `dowWeekendBuilder`.

Reply:
Just want to clarify, if I got it right:
final WeekdayBuilder weekdayBuilder; => final DowBuilder dowWeekdayBuilder;

Also add another builder for 2 weekend days from weekday row, which will override dowWeekdayBuilder for weekends, if both exist:
final DowBuilder weekendBuilder;
...